### ISSUE TITLE
Expanse GPU jobs fail with MPI error

### ISSUE THREAD

**Issue by joaander** [glotzerlab/software#120]
## Script

```
#!/bin/bash
#SBATCH --job-name="test-gpu"
#SBATCH --partition=gpu-shared
#SBATCH --nodes=1
#SBATCH --ntasks-per-node=1
#SBATCH --gpus=1
#SBATCH -t 0:10:00

module load gpu singularitypro openmpi/4.0.4

singularity exec --nv software.sif python3 -c "import hoomd; device=hoomd.device.GPU(); print(device.devices)"
```

## Output

```
A requested component was not found, or was unable to be opened.  This
means that this component is either not installed or is unable to be
used on your system (e.g., sometimes this means that shared libraries
that the component requires are unable to be found/loaded).  Note that
Open MPI stopped checking at the first component that it did not find.

Host:      exp-2-60
Framework: btl
Component: openib
```

## Expected output

```
['[0]  Tesla V100-SXM2-32GB  80 SM_7.0 @ 1.53 GHz, 32510 MiB DRAM']
```

## Configuration

```beta-expanse```

Expanse.

**Comment by joaander** [comment]
Expanse appears to have different MPI configurations for the GPU and CPU nodes.

**Comment by joaander** [comment]
Fixed in #125.
